scripts/copy-script.sh

```diff
@@ -2,7 +2,7 @@
 set -u
 
 SCRIPT=$1
-BUILD_SCRIPT=build-script
+BUILD_SCRIPT=/workspace/build-script
```

Comment:
for some reason /build-script works with Tekton v0.16.3 but not with latest version.  Hence, creating the build-script in /workspace works for both